Game/Config/DefaultGame.ini

```diff
@@ -16,3 +16,12 @@ MaxSpectators=1000
 
 [/Script/GameplayAbilities.AbilitySystemGlobals]
 +GameplayCueNotifyPaths="/Game"
+
+[Staging]
++WhitelistConfigFiles=GDKTestGyms/Config/DefaultSpatialGDKEditorSettings.ini
++WhitelistConfigFiles=GDKTestGyms/Config/DefaultSpatialGDKSettings.ini
++WhitelistConfigFiles=GDKTestGyms/Config/MapSettingsOverrides/TestOverridesBase.ini
++WhitelistConfigFiles=GDKTestGyms/Config/MapSettingsOverrides/TestOverridesOneClientGroup.ini
++WhitelistConfigFiles=GDKTestGyms/Config/MapSettingsOverrides/TestOverridesSpatialEventTracingGroup.ini
++WhitelistConfigFiles=GDKTestGyms/Config/MapSettingsOverrides/TestOverridesSpatialSnapshotTestPart1Map.ini
++WhitelistConfigFiles=GDKTestGyms/Config/MapSettingsOverrides/TestOverridesSpatialSnapshotTestPart2Map.ini
```

Comment:
I think we already discussed this, but can't exclude an entire directory, right?

Reply:
Spoke to @mironec offline. This is a nice idea but unfortunately the whitelist only accepts absolute paths to files.